liboptv/tests/check_segmentation.c

```diff
@@ -6,62 +6,73 @@
 
 
 START_TEST(test_peak_fit)
-{
-    int ntargets; 
-    unsigned char img[5][5] = {
```

Comment:
Just define it as ``unsigned char img[25]`` and remove the inner curly braces. No need for a temp and a copy.